__tests__/TreeNode.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import TreeNode from '../src/components/TreeNode';
+import defaultTheme from '../src/themes/default';
+import defaultAnimations from '../src/themes/animations';
+import defaultDecorators from '../src/components/Decorators';
+import data from '../example/data';
+
+describe('<TreeNode/>', () => {
+    describe('when onToggle function is called', () => {
+        test('should call correctly', () => {
+            const clickFn = jest.fn();
+            const wrapper = shallow(<TreeNode
+                node={data}
+                decorators={defaultDecorators}
+                animations={defaultAnimations}
+                style={defaultTheme}
+                onToggle={clickFn}
+            />);
+            wrapper.find('#button').simulate('click');
```

Comment:
you can declare a function to avoid the hardcoded id be copy-pasted several times on this file:
```javascript
const pageObjectModel = wrapper => ({
  button: () => wrapper.find('#button')
});
```